src/util/helpers.tsx

```diff
@@ -426,6 +426,10 @@ export const sortByOptionsDesc = (a: Record<string, any>, b: Record<string, any>
   return sortStringDesc(valA, valB);
 };
 
+export const sortBySortOrderAsc = (a: Record<string, any>, b: Record<string, any>): number => {
+  return sortNumberByKeyAsc(a, b, 'sort_order');
```

Comment:
What is going to happen when `sort_order` is not set, as it can't be set from the form editor (like mentioned in the API PR). The default value seems to be 0, so if there was any ordering before, isn't this going to invalidate it and make it random?